BlueprintUI/Tests/ElementContentTests.swift

```diff
@@ -70,7 +70,80 @@ class ElementContentTests: XCTestCase {
             container.measure(in: SizeConstraint(CGSize.zero), environment: .empty),
             CGSize(width: 600, height: 800))
     }
-    
+
+    func test_cacheTree() {
```

Comment:
Worth writing a test to ensure that for a given size constraint during a full layout pass, the actual measure function is only called once? Eg, to verify the original bug that caused re-measurements due to the singleton key not being used in all cases.